src/index.js

```diff
@@ -74,3 +76,5 @@ deleteBtn.addEventListener('click', () => {
 });
 
 renderTasks();
+readTask();
+updateTask();
```

Comment:
- [ ] You have done a great job so far but there are still some requirements you need to meet before moving to the next milestone. :+1: :100: 
 - - As pointed out by the previous reviewer the checkbox only updates the current state of the UI. You also need to update the completed status of the todos in the local storage so that even if a user refreshes, The status of each todo is maintained. :smile: :+1: 
 
 - - Also as the previous reviewer suggested, the `Clear all completed` button should delete all todos marked as completed, But currently, it doesn't do that. I'll suggest you go over your code base one more time and implement this functionality as its part of the project requirement. :wave: :smile: